### ISSUE TITLE
Stage I

### ISSUE THREAD

**Issue by ije** [alephjs/alef-component-draft#3]

**Comment by shadowtime2000** [comment]
@ije Will the compiler allow using a ternary operator instead of an if statement for a conditional? I find it much more concise

**Comment by ije** [comment]
@shadowtime2000 yes, please ref #8 , i also think the top `if-else` is kind of weird...

**Comment by ije** [comment]
if you are rust user, it is vary familiar. but in JS i am not sure...i'm trying to extend the conditional rendering that is weak in JSX syntax.
